package.json

```diff
@@ -14,6 +14,7 @@
   },
   "dependencies": {
     "@hot-loader/react-dom": "^17.0.2",
+    "@medv/finder": "^2.1.0",
```

Comment:
Wasn't this already supposed to be in a previous PR of @MariyaShavshy ?

Reply:
yes, it was. But after I followed the instruction of her to install the packages, it looks like mine was more recent version of the package. And it seems like updating the file as well.